nodecontrol/NodeController.go

```diff
@@ -129,7 +131,7 @@ func killPID(pid int) error {
 		}
 		select {
 		case <-waitLong:
-			return syscall.Kill(pid, syscall.SIGKILL)
+			return util.KillProcess(pid, syscall.SIGTERM)
```

Comment:
Unless there is a good reason to change the SIGKILL to SIGTERM, I'd like to stick with the existing logic and keep it with SIGTERM. SIGTERM can be caught and ignored, whereas SIGKILL cannot.

Reply:
I saw it tonight and was about to fix it.